gbus/tx/sagastore.go

```diff
@@ -37,8 +38,9 @@ func (store *SagaStore) scanInstances(rows *sql.Rows) ([]*saga.Instance, error)
 		var startedBySaga sql.NullString
 		var startedByMsgID sql.NullString
 		var startedByRPCID sql.NullString
+		var createdAt time.Time
 
-		error := rows.Scan(&sagaID, &sagaType, &sagaData, &startedBy, &startedByMsgID, &startedByRPCID, &startedBySaga, &version)
+		error := rows.Scan(&sagaID, &sagaType, &sagaData, &startedBy, &startedByMsgID, &startedByRPCID, &startedBySaga, &version, &createdAt)
```

Comment:
pay attention that instances that already existed in the db prior to this feature will have a null value in the created_at column. not sure how row.scan will work here. need to make sure it won't panic or add a value during the migration

Reply:
added default fo that column 👍🏼